Cakes-Co/style.css

```diff
@@ -1 +1,229 @@
-/*  Add your styling here */
+* {
+  margin: 0;
+  padding: 0;
+}
+body {
+  font-family: "Roboto", sans-serif;
+  font-size: 4vw;
+}
+h1 {
+  font-size: 2.4em;
+  font-weight: 500;
+}
+h4 {
+  font-size: 1.2em;
+  font-weight: 400;
+  font-style: italic;
+}
+h6 {
+  font-size: 0.7em;
+  font-weight: 200;
+}
+p {
+  font-size: 1em;
+  font-weight: 300;
+  line-height: 1.4em;
+}
+img {
+  width: 100%;
+}
+
+:root {
+  --backgroundColor: rgb(153, 177, 233);
+  --generalPadding: 1em;
+  --sectionGap: 2em;
+  --white: white;
+}
+
+header {
+  height: 40vw;
+  padding: var(--generalPadding);
+  background-color: var(--backgroundColor);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  color: var(--white);
+}
+.logo-img {
+  margin-top: var(--generalPadding);
+  height: 70%;
+  width: auto;
+}
+.header-area {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: flex-end;
+  text-align: end;
+}
+#menu-inpt {
+  display: none;
+}
+.menu-lbl {
+  height: 30%;
+  width: auto;
+  position: relative;
+  cursor: pointer;
+}
+.menu-img {
+  height: 100%;
+}
+.header-area p {
+  width: 19ch;
+}
+.nav-nav {
+  display: none;
+  width: min-content;
+  border-radius: var(--generalPadding);
+  padding: var(--generalPadding);
+  background-color: black;
+  position: absolute;
+  top: 70px;
+  right: 100%;
+  transition: 0s;
+}
+
+#menu-inpt:checked + .nav-nav {
+  display: block;
+  right: var(--generalPadding);
+}
```

Comment:
I must say, your solution is excellent! It's not only readable and easy to follow, but it also demonstrates a practical approach. Loved this one.